.gitignore

```diff
@@ -1,4 +1,5 @@
-node_modules
+server/node_modules
+client/node_modules
```

Comment:
this is fine, but just know that the plain "node_modules" would ignore any file or folder named node_modules in the source tree:
`$ git check-ignore -v client/node_modules
.gitignore:1:node_modules	client/node_modules`